rpm/0002-Drop-smartata-dependencies.patch

```diff
@@ -1,7 +1,7 @@
-From d16ccca1c503badece76d462e524c5d530fb8c64 Mon Sep 17 00:00:00 2001
+From 4be9f903c40102380fd2b115644c499a5f2aa826 Mon Sep 17 00:00:00 2001
 From: Raine Makelainen <raine.makelainen@jolla.com>
 Date: Thu, 7 Dec 2017 13:15:34 +0200
-Subject: [PATCH] Drop smartata dependencies
+Subject: [PATCH 02/13] Drop smartata dependencies
```

Comment:
Please keep on using ```--no-numbered``` / ```-N``` when formatting patches with git format-patch. Also I'd not mind if you'd take ```--zero-commit``` into use so that "From sha1" header line would stay the same when going forward.

Reply:
Done refactored patches are also in this tree:
https://github.com/Thaodan/udisks/tree/sailfishos/udisks-2.9.4